--- standalone.conf.original	2017-03-19 18:47:30.451588656 +0000
+++ standalone.conf	2017-03-19 18:50:30.787679872 +0000
@@ -49,6 +49,8 @@
 if [ "x$JAVA_OPTS" = "x" ]; then
    JAVA_OPTS="-Xms1303m -Xmx1303m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
+   JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address=JBOSS_BIND_ADDRESS"
+   JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address.management=JBOSS_BIND_ADDRESS_MANAGEMENT"
 else
    echo "JAVA_OPTS already set in environment; overriding default settings with values: $JAVA_OPTS"
 fi
